<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture Tool</string>
  </property>
  <widget class="QPushButton" name="btnCapture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>134</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>6</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>36</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>66</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl4">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>96</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>220</y>
     <width>353</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="grpCapture" native="true">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>35</y>
     <width>135</width>
     <height>47</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblSec_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>27</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>with</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSec">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>27</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sec. delay</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>7</y>
      <width>133</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="cntDelay">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>22</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="grpEdit" native="true">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>82</y>
     <width>137</width>
     <height>379</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>205</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Components:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClipboard">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>286</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>C&amp;lipboard</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>195</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>256</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Save</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>245</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnhancment">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>316</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Enhancment</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>47</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Drawing Tools:</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>37</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUndo">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>17</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>17</y>
      <width>59</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Redo</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>5</y>
      <width>133</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>149</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="btnColorSelector">
       <property name="text">
        <string>C&amp;olor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QButtonFrame" name="frm1Color" native="true">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QButtonFrame" name="frm2Color" native="true"/>
       </item>
       <item>
        <widget class="QButtonFrame" name="frm3Color" native="true"/>
       </item>
       <item>
        <widget class="QButtonFrame" name="frm4Color" native="true"/>
       </item>
       <item>
        <widget class="QButtonFrame" name="frm5Color" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QButtonFrame" name="frmCurrentColor" native="true">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>142</y>
      <width>125</width>
      <height>5</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>64</y>
      <width>63</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cntPenSize">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>62</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QButtonFrame</class>
   <extends>QWidget</extends>
   <header>qbuttonframe.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
